ruby-challenges.rb

```diff
@@ -1,36 +1,109 @@
+system('clear')
 # ASSESSMENT 4: Ruby Coding Practical Questions
 # MINASWAN ✌️
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
+def even_odd(num)
+    if num % 2 == 0
+         'num is even'
+        else 
+            'num is odd'
+    end
+end
+
 reposts1 = 7
 # Expected output: '7 is odd'
+p even_odd(reposts1)
+
 reposts2 = 42
 # Expected output: '42 is even'
+p even_odd(reposts2)
 reposts3 = 221
 # Expected output: '221 is odd'
+p even_odd(reposts3)
 
 # Pseudo code:
+# create a method called 'even_odd'
+# Will definetley contain an if/else statement 
+#use num as the placeholder where the var will go later 
+#use the modulo to apply the "even/odd logic"
+#if nots not even it must be odd << my logic for not creating an elsif for odds
+# outcome is "num is even" or "num is odd"
+
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. 
 # HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
+def no_vowels(string)
```

Comment:
Logic looks great.  Just make sure to double-check your outputs and that they match the expected output.  In this case, what happens if the vowel is capitalized?